### PR TITLE
allow success_return_status in addition to success_return_redirect

### PR DESCRIPTION
**Pull request by ghost**
see http://docs.aws.amazon.com/AmazonS3/latest/API/sigv4-HTTPPOSTForms.html for details on the success_return_status parameter.

We need this as we do our upload with ajax instead of a page reload.

### CODE REVIEW

File: _src/lib/aws-s3-form.coffee [hunk 1]
```diff
@@ -87,18 +89,22 @@ class AwsS3Form extends require( "mpbasic" )()
 			_secure = @config.secure
 
 		data =
-			action: "#{ if _secure then "https" else "http" }://#{ @config.bucket }.s3.amazonaws.com/"
+			action: "#{ if _secure then "https" else "http" }://s3-#{@config.region}.amazonaws.com/#{ @config.bucket }"
```
Comment (ghost) [inline _src/lib/aws-s3-form.coffee:93]: changed the url so that the s3 wildcard cert would be valid.

**Comment by ghost** [comment]
closing this PR as I couldn't quite get it to work, but I'll leave the branch if you want to take a shot.

**Comment by mpneuried** [comment]
Thanks for your pull-request.
I will have a look at you changes with `@_successActionStatus()` the next days.

**Comment by mpneuried** [comment]
Hi @tmacie,
i had a look at your code and adapted it to version `0.2.0`.
Now you can define a option `successActionStatus` witch will replace the redirect.
Within the example i added a Parameter-Element called "StatusCode-Policy" to test it.

Is this a suitable solution for you?

**Comment by ghost** [comment]
Yup, thats perfect!  Thanks for the fix.

**Pull request closed without merging**